app/templates/components/topics/list-container.hbs

```diff
@@ -0,0 +1,5 @@
+<ul class="topics">
+  {{#each topics as |topic|}}
+    <li>{{topics/list-item topic=topic}}</li>
```

Comment:
ha, I wasn't even thinking it was named after li.  I was thinking it was literally an item in the list of topics.